lib/tasks/data_hygiene.rake

```diff
@@ -40,4 +40,21 @@ namespace :data_hygiene do
   task :bulk_update_organisation, %i[csv_filename] => :environment do |_, args|
     DataHygiene::BulkOrganisationUpdater.call(args[:csv_filename])
   end
+
+  desc "Discard drafts (if present) and unpublish"
+  task :discard_drafts_and_unpublish, %i[content_id] => :environment do |_, args|
+    document = Document.where(content_id: args[:content_id]).first
```

Comment:
You could get multiple documents returned here, since the same content ID is used for multiple locales.

I'd recommend returning all of them, then looping through to discard the drafts and unpublish each one.

Reply:
How does the unpublish work with locales? It's not clear to me how to pass a document object to Commands::V2::Unpublish (the examples I can find either pass in a content id, which obviously doesn't help, or an edition). Do I discard all the drafts and then pass in the Document's latest edition?

Reply:
The discard and unpublish commands can both take a locale as an argument ([example](https://github.com/alphagov/publishing-api/pull/3047/files)).